index.ts

```diff
@@ -156,6 +156,7 @@ const LOG = {
   DEPLOYMENT_LIST: (scriptId: string) => `Listing deployments for ${scriptId}...`,
   DEPLOYMENT_START: (scriptId: string) => `Deploying project ${scriptId}...`,
   FILES_TO_PUSH: 'Files to push were:',
+  FINDING_SCRIPTS: 'Looking for your scripts now...',
```

Comment:
Pretty good, but I think we can be more terse.
Can you write:
'Finding your scripts...'